opcua/server/history_interface.py

```diff
@@ -0,0 +1,62 @@
+class HistoryStorageInterface(object):
+
+    """
+    Interface of a history backend.
+    Must be implemented by backends
+    """
+
+    def new_historized_node(self, node, period, count=0):
+        """
+        Called when a new node is to be historized
+        Returns None
+        """
+        raise NotImplementedError
+
+    def save_node_value(self, node, datavalue):
+        """
+        Called when the value of a historized node has changed and should be saved in history
+        Returns None
+        """
+        raise NotImplementedError
+
+    def read_node_history(self, node, start, end, nb_values):
```

Comment:
Sorry I forgot to change that back. Before I was using the node name for table naming. Now I only use the nodeid so I will change it back.